kernel/pre_env.mli

```diff
@@ -47,7 +47,7 @@ type named_context_val = private {
 
 type env = {
     env_globals       : globals;
-    env_named_context : named_context_val;
+    env_named_context : named_context_val;  (** in the interactive theorem proving mode, we see these above ============================ *)
```

Comment:
Commit removed.

What is the right place to explain the meaning of `Pre_env.env`?